src/modules/dashboard/components/Dialogs/ManageWhitelistDialog/ManageWhitelistDialog.tsx

```diff
@@ -51,85 +66,126 @@ const ManageWhitelistDialog = ({
   }, [setShowInput, setFormSuccess]);
 
   const history = useHistory();
+
+  /* We don't close the dialog after submitting data
+  => need a way for a refreshed colony to be reflected */
+  const { data: colonyData } = useColonyFromNameQuery({
+    variables: { name: colonyName, address: colonyAddress },
+  });
```

Comment:
Where are you suggesting to use refetch? Not clear to me...

Reply:
@chinins i just read your comment, and im not sure what exactly is the problem, but when you call refetch, data should be reloaded

Reply:
I don't understand what problem you are trying to solve by refetch - you are not clear in your comments.

Reply:
"=> need a way for a refreshed colony to be reflected */" so can you expand more this problem?

Reply:
I don't have a problem here,  it's solved by the code, I'm just commenting the code - why I use the query here, because it's not so straight forward to understand.
That's why I don't understand your suggestion of using refetch...